<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewRoomMate</class>
 <widget class="QDialog" name="NewRoomMate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Novo Morador</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Image.qrc">
    <normaloff>:/icons/img/svg/businessman.svg</normaloff>:/icons/img/svg/businessman.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="salvarMorador">
     <property name="text">
      <string>Salvar</string>
     </property>
     <property name="icon">
      <iconset resource="Image.qrc">
       <normaloff>:/icons/img/svg/checkmark.svg</normaloff>:/icons/img/svg/checkmark.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="cancelarNovoMorador">
     <property name="text">
      <string>Cancelar</string>
     </property>
     <property name="icon">
      <iconset resource="Image.qrc">
       <normaloff>:/icons/img/svg/cancel.svg</normaloff>:/icons/img/svg/cancel.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="nomeMorador"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Image.qrc"/>
 </resources>
 <connections/>
</ui>
